cli/Cargo.toml

```diff
@@ -52,10 +52,9 @@ source-map-mappings = "0.5.0"
 sys-info = "0.5.8"
 tempfile = "3.1.0"
 termcolor = "1.0.5"
-tokio = "0.1.22"
+tokio = { version = "0.2.1", features = ["full"] }
```

Comment:
After landing this PR we might want to take a look if we can strip down `full` to concrete features